hera_cal/apply_cal.py

```diff
@@ -106,7 +109,12 @@ def apply_cal(data_infilename, data_outfilename, new_calibration, old_calibratio
         uvd.flag_array = np.logical_or(npz_flags['flag_array'], uvd.flag_array)
         add_to_history += ' FLAGS_NPZ_HISTORY: ' + str(npz_flags['history']) + '\n'
     data, data_flags = io.load_vis(uvd)
-    
+
+    # apply band edge flags
+    for bl in data_flags.keys():
+        data_flags[bl][:,0:flag_nchan_low] = True
+        data_flags[bl][:,-flag_nchan_high:] = True
```

Comment:
setting `flag_nchan_high=0` will cause all data to be flagged here.

Reply:
Good call. I'll fix this.